config/jobs/mirror-images/postsubmits-mirror-images.yaml

```diff
@@ -0,0 +1,24 @@
+postsubmits:
+  rajaskakodkar/tanzu-test-infra:
+  - name: postsubmit-mirror-upstream-images-to-ecr
+    cluster: test-jobs
+    decorate: true
+    max_concurrency: 1
+    branches:
+    - ^main$
+    run_if_changed: '^scripts/mirror-images/source-images.yaml'
```

Comment:
suggestion on the path of the input file:
- It should be organised in the this test-infra repo based on github org/repo or function

e.g. 
- For Prow
   - `services/image-mirroring/prow/prow.yaml`
 - For TCE
    - `services/image-mirroring/vmware-tanzu/community-edition/tce.yaml`
    

long-term, we'll have to put one single pattern of `org/repo/release_name` for organising the image mirroring config in this repo and the prow mirroring should live elsewhere (if at all required).

Reply:
> Following the discussion from the Prow Sync-up call, resolving this in favor of:
>
> I'll work on extending the current image mirroring mechanism (for use-cases like image promotion) in a separate step.

Same!

Reply:
Keeping both the comments as pointers for the next iteration. Thank you!